homeassistant/helpers/entity_platform.py

```diff
@@ -225,11 +225,6 @@ def add_entities(self, new_entities, update_before_add=False):
         await asyncio.wait(tasks, loop=self.hass.loop)
         self.async_entities_added_callback()
 
-        if self._async_unsub_polling is not None or \
```

Comment:
You should maybe keep the first condition so that there aren't multiple poll timers around. 

https://github.com/tadly/home-assistant/blob/9fcb1697ec7117280a319f635ac0b28176deb6a4/homeassistant/helpers/entity_platform.py#L377

And also change this

Reply:
Yeah, I've been rushing that to much :/